MapboxCoreNavigationTests/NavigationServiceTests.swift

```diff
@@ -208,7 +208,7 @@ class NavigationServiceTests: XCTestCase {
                                                                     altitude: 0,
                                                                     horizontalAccuracy: 30,
                                                                     verticalAccuracy: 10,
-                                                                    course: -finalHeading,
```

Comment:
I’m surprised we got away with this until now, but negating a `CLLocationDegrees` makes it invalid rather than pointing it in the other direction.